example/tests/test_model_viewsets.py

```diff
@@ -185,6 +185,24 @@ def test_patch_requires_id(self):
 
         self.assertEqual(response.status_code, 400)
 
+    def test_patch_requires_correct_id(self):
+        """
+        Verify that 'id' is required to be passed in an update request.
+        """
+        data = {
+            'data': {
+                'type': 'users',
+                'id': 3,
```

Comment:
```suggestion
                'id': self.miles.pk + 1,
```

it could be that self.miles.pk is actually three... so this to be safe.